web/packages/teleport/src/TopBar/TopBar.tsx

```diff
@@ -107,30 +109,57 @@ export function TopBar({ CustomLogo }: TopBarProps) {
               }
             `}
           >
-            <NavigationButton
-              selected={resourceTabSelected}
-              to={cfg.getUnifiedResourcesRoute(clusterId)}
-              title="Resources"
-            >
-              <Server
-                color={resourceTabSelected ? 'text.main' : 'text.muted'}
-              />
-              <Text
-                ml={3}
-                fontSize={18}
-                fontWeight={500}
-                css={`
-                  display: none;
-                  @media screen and (min-width: ${p =>
-                      p.theme.breakpoints.medium}px) {
-                    display: block;
-                  }
-                `}
-                color={resourceTabSelected ? 'text.main' : 'text.muted'}
+            {cfg.isDashboard ? (
+              <NavigationButton
+                selected={downloadTabSelected}
+                to={cfg.routes.downloadCenter}
+                title="Downloads"
               >
-                Resources
-              </Text>
-            </NavigationButton>
+                <Download
+                  color={downloadTabSelected ? 'text.main' : 'text.muted'}
+                />
+                <Text
+                  ml={3}
+                  fontSize={18}
+                  fontWeight={500}
+                  css={`
+                    display: none;
+                    @media screen and (min-width: ${p =>
+                        p.theme.breakpoints.medium}px) {
+                      display: block;
+                    }
+                  `}
+                  color={downloadTabSelected ? 'text.main' : 'text.muted'}
+                >
+                  Downloads
+                </Text>
```

Comment:
i'd create a re-usable component (that accepts `selected`), since we could change the font size and font weight later (i could see us missing this one)

Reply:
For sure, thanks, added a component and reused for the three main nav items.